platform/sql-scripts/init.sql

```diff
@@ -0,0 +1 @@
+CREATE DATABASE IF NOT EXISTS `kbb_db`;
```

Comment:
Is this being used?

Reply:
Yes. Because tekai's python script did not include db creation.

Reply:
Sorry, can show me where is it being used? Because I need to think how to initialise the prod database. 

Now the pipeline is just building and push the image to ECS. Need to run the database_object python somewhere